@xen-orchestra/web/src/stores/xo-rest-api/pif.store.ts

```diff
@@ -32,19 +32,25 @@ export const usePifStore = defineStore('pif', () => {
     return hostMasterPifsByNetworkMap
   })
 
-  const pifsByNetwork = computed(() => {
-    const pifsByNetworkMap = new Map<XoNetwork['id'], XoPif[]>()
+  const getPifsByNetworkRef = (networkRef: XoNetwork['id']) => {
```

Comment:
Maybe `networkId` would be more suitable? In XO 6 we're dealing with `id` and not `ref` like in XO Lite.
```suggestion
  const getPifsByNetworkRef = (networkId: XoNetwork['id']) => {
```